app/controllers/concerns/hal_actions.rb

```diff
@@ -94,8 +93,23 @@ def resources_name
     resource_name.pluralize
   end
 
-  def with_params(keys, arel)
-    keys ||= []
+  def decorate_query(res)
+    res = with_sorting(res)
+    res = with_paging(res)
+    res = with_params(res)
+    res
+  end
+
+  def with_sorting(arel)
+    arel.order(id: :desc)
```

Comment:
it is broken out so sorting can be specified without overriding `resources`.

as to the default, I hear ya, it was already defaulting to this, so this is not a change just moving where it happens to a better place:
``` ruby
# previously it did this
res = self.class.resource_class.order(id: :desc).page(params[:page])
```
https://github.com/PRX/cms.prx.org/pull/67/files#diff-05e2417b36696821063bd3631c843187L88

I was questioning it when I added it, but still lean to having it add something.

I would not be against changing it to do nothing by default (return the argument), but by id: :desc seemed like a more common case and so a more logical default.